### ISSUE TITLE
[Tai Nguyen] Peer Review week 3

### ISSUE THREAD

**Issue by TaiNguyen2407** [pennane/chess#3]
Your README file is comprehensive, providing clear instructions and a roadmap that outlines the project's direction. The commit messages are informative, following commit convention standard. The project's structure is commendable, with an organization of files into components folders such as "chess" and "ui."

Most of the functions are named descriptively. In summary, the project shows a high standard of quality and a solid foundation with an great project structure, clear naming conventions, and informative commit messages.

The only concern I have is some functions might need a bit more informative name (e.g startComputerOnlyGame), otherwise great work for the project

**Comment by pennane** [comment]
Yeah, some of the functions are a mess. I will try to improve them. It is quite a challenge to abstract technical problems down into something readable.
